models/order.js

```diff
@@ -1,6 +1,7 @@
 import mongoose from 'mongoose';
 import { Schema } from 'mongoose';
 import constants  from '../src/constants';
+import Item from './item';
 
 const Entry = new Schema({
```

Comment:
you mean move 'getOrder' to another file? Since it's part of an Order model, I figured it made sense to put it here.